posthog/cdp/templates/mailgun/template_mailgun.py

```diff
@@ -0,0 +1,106 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+
+# See https://documentation.mailgun.com/docs/mailgun/api-reference/openapi-final/tag/Messages
+
+
+template_mailgun_send_email: HogFunctionTemplate = HogFunctionTemplate(
+    status="alpha",
+    id="template-mailgun-send-email",
+    name="Send an email via Mailgun",
+    description="Send emails using the Mailgun HTTP API",
+    icon_url="/static/services/mailgun.png",
+    hog="""
+if (empty(inputs.template.to)) {
+    return false
+}
+
+fn multiPartFormEncode(data) {
+    let boundary := f'---011000010111000001101001'
```

Comment:
might be worth generating a random uuid here?

Reply:
yeah i originally did but it didn't work for some reason

Reply:
The spec says it has to be a specific length i think